pkg/env/v1alpha1/env.go

```diff
@@ -51,6 +51,12 @@ const (
 	// This environment variable is set via kubernetes downward API
 	OpenEBSMayaPodName ENVKey = "OPENEBS_MAYA_POD_NAME"
 
+	// CSPCOperatorPodName is the environment variable to get cspc-operator pod
+	// name
+	//
+	// This environment variable is set via kubernetes downward API
+	CSPCOperatorPodName ENVKey = "CSPC_OPERATOR_POD_NAME"
```

Comment:
Let us put this somewhere in the cspc operator app package

Reply:
This will require a few other changes too that are out of the scope of the PR.
( The function `env.Get` arg type is `ENVKey` which is defined in the package)